libs/island-ui/core/src/lib/PhoneInput/CountryCodeSelect/CountryCodeSelect.tsx

```diff
@@ -34,27 +36,27 @@ export const CountryCodeSelect = ({
   onBlur,
   onMenuOpen,
   onMenuClose,
-}: CountryCodeSelectProps & TestSupport) => {
+}: CountryCodeSelectProps) => {
   return (
     <div
       className={cn(styles.wrapper, {
-        [styles.wrapperColor[backgroundColor]]: !disabled,
-        [styles.containerDisabled]: disabled,
+        [styles.wrapperColor[backgroundColor]]: !isDisabled,
+        [styles.containerDisabled]: isDisabled,
       })}
       data-testid={`country-code-select-${name}`}
     >
-      <ReactSelect
+      <ReactSelect<OptionType<string>, false, GroupBase<OptionType<string>>>
```

Comment:
This looks weird, is this overriding the props that are being passed to <ReactSelect ?